### PR TITLE
addpkg(x11/xcape)

### PR DESCRIPTION
**Pull request by babaric-dev**

### CODE REVIEW

**Comment by TomJo2000** [comment]
Let's check that against the package request template.

---

### Why is it worth to add this package?

Let's modifier keys act as standalone keys.

### Home page URL

See PR

### Source code URL

See PR

### Packaging policy acknowledgement

- [X] The project is actively developed.
- [X] The project has [existing packages](https://repology.org/projects) and is "well known".
- [X] Licensed under an [open source license](https://spdx.org/licenses/).
- [X] Not available through a language package manager: pip, npm, cpan, cargo, etc.

- [X] Not taking up too much disk space (< 100MiB per architecture, exceptions can be made)
- [X] Not duplicating the functionality of existing packages.
- [X] Not serving hacking, malware, phishing, spamming, spying, ddos functionality.
- [X] I certify that I have read [Termux Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy) and understand that my request will be denied if it is found lacking.

### Additional information

N/A

---

Looks good to me.

File: x11-packages/xcape/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/alols/xcape
+TERMUX_PKG_DESCRIPTION="Linux utility to configure modifier keys to act as other keys when pressed and released on their own."
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.2
+TERMUX_PKG_SRCURL=https://github.com/alols/xcape/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=a27ed884fd94f03058af65a39edfe3af3f2f8fbb76ba9920002a76be07fb2821
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_BUILD_DEPENDS="libx11,libxtst"
```
Comment (TomJo2000) [inline x11-packages/xcape/build.sh:10]: Neither of the dependencies are in the X11 repo, so please put this package in the main `packages` repo as well, unless there is an unstated reason this package depends on a graphical output.

**Review by TomJo2000 — CHANGES_REQUESTED** [review]

File: x11-packages/xcape/build.sh (hunk 1 above)
Comment (babaric-dev) [reply x11-packages/xcape/build.sh:10]: those are build dependencies

Comment (TomJo2000) [reply x11-packages/xcape/build.sh:10]: And that means the package needs to go in the X11 repo?

Comment (babaric-dev) [reply x11-packages/xcape/build.sh:10]: it needs x11.

```
$ DISPLAY= xcape
Unable to connect to X11 display. Is $DISPLAY set?
```

Comment (TomJo2000) [reply x11-packages/xcape/build.sh:10]: Ah okay, thanks for the confirmation.

**Pull request merged**